vignettes/O4_make_a_package.Rmd

```diff
@@ -19,13 +19,13 @@ library(rUM)
 ```
 
 ## Introduction
-Saving your data and analyses into an R project helps to create reproducible research.  Someone who wants to reproduce your work can load a package to have access to your data and your paper(s).  rUM can help you buy creating a project that has all the files/directories needed for an R package including a vignette folder that contains the outline of a paper.
+Saving your data and analyses into an R package helps to create reproducible research.  Someone who wants to reproduce your work can load a package to have access to your data and your paper(s).  **`rUM`** can help you by creating a project that has all the files/directories needed for an R package including a vignette folder that contains the outline of a paper.
 
 ## How do I make a package project?
-All you need to do is add `vignette = TRUE` when you use the `make_project()` function and then once the project is created run the **run_me.R** file.  For example type:
+All you need to do is add `vignette = TRUE` when you use the `make_project()` function. Once the project is created, run the **run_me.R** file.  For example type:
 
-1. `make_project("~/Desktop/my_example", "R", example=TRUE, vignette=TRUE)`.  This makes a new project with all the files for a package in the current directory.  The vignette includes an example table and figure and a the R Markdown syntax for adding hyperlinks to the table and figures.
-2. After the new project is open, run the **read_me.R** file.  If you are working in RStudio it will ask you if it can/should close windows for two delete two files.  Say **Yes**. They are a couple temporary files that we used to set options the project.
+1. `make_project("~/Desktop/my_example", "R", example=TRUE, vignette=TRUE)`.  This creates a new project with all the files for a package in the current directory.  The vignette includes an example table and figure and all the R Markdown syntax for adding hyperlinks to the table and figures.
+2. After the new project is open, run the **read_me.R** file. The easiest way to do this is by typing `source(run_me.R)` into the RStudio IDE console and then pressing enter. If you are working in RStudio it will ask you if it can/should close windows for two deleted files.  Say **Yes**. They are a couple temporary files that we used to set options the project.
```

Comment:
CORRECTION: this needs to be changed to `source("run_me.R")`... the quotation marks were not added by mistake